### PR TITLE
Release 0.1.13

### PR DESCRIPTION
**Pull request by mradkov**
# 0.1.13 (2020-08-17)

## Release highlights

This release introduces an integrated wallet in the right section of the superhero UI. It allows users to start using the superhero platform without having an extension installed or go switching back and forth between tabs to confirm signatures of actions in their web wallet.

Some improvements to the web-wallet communication were put in place as well.

### Bugfixes 🐞 
- Display whole sized search icon (#718)
- Hotfix meet bug (#722)
- TipComments: Get rid of infinite fetch loop (#736)

### Features ✨ 
- Integrated Web-Wallet (#528)

### Maintenance 🚧 
- TipRecord: Avoid extra rerendering (#712)
- TipRecord: Simplify condition (#711)
- Add on Aepp disconnect event (#710)
- Avoid wrapping of RouterView with KeepAlive (#708)
- Refactor layout components for the integrated wallet support (#720)
- Refreshed integrate wallet into UI (#721)
- Maintenance page fixes (#726)
- Refactor TipList page (#727)
- Dispose jitsi instance after page leave (#730)
- Refactor TipInput and remove unused commenting functionality (#732)
- Refactor Overview component (#735)
- Extract MessageInput component (#751)
- Remove Wizard components completely (#749)
- Update wallet url .env (#745)
- Avoid usage of Page component (#748)
- UserProfile: Update data on address change (#754)
- Rename state.account to state.address (#753)
- Remove useless navigation guards and initial 404 behavior (#755)

### Other changes💡
- Set pinning and reporting to work with web-wallet (#717)
- Keep RightSectiopnTopics open if no iframed wallet (#741)
- SendTip: fix icon size (#731)
- Move Meet up in the menu (#752)
- FAQ, CreateProfile: Clean up translations (#750)
- Chinese translation update (#744)
- French Translation (#743)

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1,2 +1,2 @@
-VUE_APP_WALLET_URL=https://wallet.superhero.com
-VUE_APP_JITSI_URL=test.league.aeternity.org
+VUE_APP_WALLET_URL=https://develop.wallet.z52da5wt.xyz
```
Comment (davidyuk) [inline .env:1]: I think we should release the wallet firstly

Comment (mradkov) [reply .env:1]: https://github.com/aeternity/superhero-ui/pull/745

File: src/components/layout/Navigation.vue [hunk 2]
```diff
@@ -1,56 +1,43 @@
...
+      {{ $t('components.layout.Navigation.League') }}
+    </RouterLink>
+    <RouterLink :to="{ name: 'voting' }">
+      <IconGovernance />
+      {{ $t('components.layout.Navigation.Governance') }}
+    </RouterLink>
+    <RouterLink :to="{ name: 'conference' }">
+      <IconMeet />
+      {{ $t('Networking.MenuLink') }}
+    </RouterLink>
```
Comment (mradkov) [inline src/components/layout/Navigation.vue:36]:
```suggestion
//    <RouterLink :to="{ name: 'conference' }">
//      <IconMeet />
//      {{ $t('Networking.MenuLink') }}
//    </RouterLink>
```

File: src/components/layout/Navigation.vue [hunk 3]
```diff
@@ -1,56 +1,43 @@
...
+      {{ $t('components.layout.Navigation.League') }}
+    </RouterLink>
+    <RouterLink :to="{ name: 'voting' }">
+      <IconGovernance />
+      {{ $t('components.layout.Navigation.Governance') }}
+    </RouterLink>
+//    <RouterLink :to="{ name: 'conference' }">
+//      <IconMeet />
+//      {{ $t('Networking.MenuLink') }}
+//    </RouterLink>
```
Comment (mradkov) [inline src/components/layout/Navigation.vue:36]:
```suggestion
    <!-- <RouterLink :to="{ name: 'conference' }">
      <IconMeet />
      {{ $t('Networking.MenuLink') }}
    </RouterLink> -->
```

**Pull request merged**